src/middleware/auth.middleware.js

```diff
@@ -0,0 +1,16 @@
+const jwt = require('jsonwebtoken');
+const { getUserWithId } = require('../repository/users.repostitory')
+
+const auth = async (req, res, next) => {
+    try {
+        const token = req.header('Authorization').replace('Bearer ', '');
+        const decoded = jwt.verify(token, process.env.JWT_SECRET);
```

Comment:
please update readme.md to remind developer to setup `process.env.JWT_SECRET` in their env file

Reply:
Solved in  b09bae5